dvc.lock

```diff
@@ -15,4 +15,4 @@ run_benchmarks:
     md5: 9531dcc1c1aa3feb29c21b152f2b4a5c
   outs:
   - path: results
-    md5: b43da8b8861285e678e40217e5959779.dir
+    md5: d751713988987e9331980363e24189ce.dir
```

Comment:
reset results, previous results seem to be unreliable, as I am not reproducing on all commits sha-s, I don't want to mix previous and current results